### ISSUE TITLE
Min value for img2img strength seems too high in latest version?

### ISSUE THREAD

**Issue by jmilr** [godly-devotion/MochiDiffusion#266]
### Running Latest Version

- [X] I am running the latest version

### Processor

M1 (or later)

### Memory

8GB

### What happened?

I get results very different to the original image even when set to minimum (e.g. for a final pass after manual tweaking in Photoshop etc.) - is there some way to get more granular control?

**Comment by jrittvo** [comment]
Can you install Mochi 3.2  for a little while to see if it gives you better control?  The range of image2image settings was changed in Mochi 4.0 because most people found that any setting of the slider from the middle to the left did not have any effect at all in moving the generated image away from the starting image.  If that change was a mistake, it can be reverted, but your experience contradicts our testing.  Setting the slider all the way to the left should pretty much ignore the text prompt and just duplicate the image, even in Mochi 4.0.

**Comment by jrittvo** [comment]
I just did a test and you may be right.  It needs a little bit lower value at the low end.  If you can please try with Mochi 3.2, though, and see where the slider starts to have an effect for you, you can help us calibrate to a better level at the low end.

**Comment by jrittvo** [comment]
This is what I get with Mochi 3.2.  The top left is the original image.  The top middle is the slider at 0 (all the way to the left).  Top right is at about 1.  Middle row goes 2, 3, 4.  Bottom row is 5, 6. 7.  

The minimum is presently at 6.7 if I remember right.  It should probably be moved down to 4.0.  Would that be better for you?

![Screencap](https://github.com/godly-devotion/MochiDiffusion/assets/114269461/bd63cfbb-7b1f-442f-abf2-8ae36800f82f)

**Comment by jmilr** [comment]
So I'm not crazy!  - the minimum went from 0 to 6.7? Here's what I get in 4.0, without any prompt, just img2img set to lowest, and with a basic prompt
<img width="1046" alt="Screenshot 2023-06-20 at 2 07 07 pm" src="https://github.com/godly-devotion/MochiDiffusion/assets/15384770/cbf9b874-e656-4ba4-9d00-2fc47c72a081">

**Comment by jrittvo** [comment]
Yep.  That matches what I see in the pics I posted above from 3.2.  The change just begins with the slider at around 4.  Moving the minimum from 0 to 6+ was too far.  A minimum of 4 would be better.  I actually did a fair amount of testing to try to pin down the minimum value that made any change, but I blew it.  Sorry!

**Comment by jrittvo** [comment]
And you're one of the first people I've seen using one of the ControlNet capable base models.  Is there a model you'd like converted to handle ControlNet that is not one of the 6 done already?  I'd be happy to add one for you.

**Comment by jmilr** [comment]
No need to apologise - people use SD to be creative, I'm using it to be lazy - it's an outlier issue 😅

In terms of models for CN, I don't really have any favourites, just tend to flick through the Core ML ones that are available until I get something I like. Thanks for the offer though!

**Comment by jrittvo** [comment]
This is now set to a lower minimum value to allow preserving more of the starting image.

**Comment by godly-devotion** [comment]
Fixed via 2f8134d8d635bf0f0e244ac8ed24fdc7d1140731
